hack/teardown.sh

```diff
@@ -6,6 +6,7 @@ source "$(dirname "${BASH_SOURCE[0]}")/lib/__sources__.bash"
 set -Eeuo pipefail
 
 debugging.setup
+dump_state.setup
```

Comment:
should we leave it?

That said... I am fine to remove in a separate PR, in case we feel of removing this (but, sure we can keep it)